app/model/Upload.scala

```diff
@@ -0,0 +1,16 @@
+package model
+
+import org.cvogt.play.json.Jsonx
+import play.api.libs.json._
+
+case class UploadPolicy(bucket: String, key: String, region: String, credentials: UploadCredentials)
+case class UploadCredentials(temporaryAccessId: String, temporarySecretKey: String, sessionToken: String)
+
+object UploadPolicy {
+  implicit val format: Format[UploadPolicy] = Jsonx.formatCaseClass[UploadPolicy]
```

Comment:
Why Jsonx? I googled the import (https://github.com/xdotai/play-json-extensions) and it looks to be only needed for case classes with 22+ parameters which this doesn't have?